src/gengowatcher/config.py

```
@@ -584,14 +584,24 @@ def _serialize_toml_value(value: Any) -> str:
             return '""'
-        return AppConfig._serialize_toml_value(str(value))
-
-        try:
-            return json.dumps(value)
-        except (TypeError, ValueError):
-            pass
+        if isinstance(value, bool):
+            return "true" if value else "false"
+        if isinstance(value, (int, float)):
+            return str(value)
+        return json.dumps(value)
```

Ruff (0.15.10)

[error] Undefined name `json`

(F821)